examples/agentInteractionsDemo/app.js

```diff
@@ -152,19 +152,27 @@ new Vue({
     },
 
     beforeMount() {
-        let pcEnvironment = getEmbeddingPCEnv();
-        if (!pcEnvironment) {
-            this.errorMessage = 'Cannot identify App Embeddding context.  Did you forget to add pcEnvironment={{pcEnvironment}} to your app\'s query string?';
-            return;
-        }
+        const queryParameters = getQueryParameters();
+        const state = computeState(queryParameters);
+        let pcEnvironment = queryParameters.pcEnvironment;
 
         let client = platformClient.ApiClient.instance;
         let clientApp = null;
         try {
-            clientApp = new window.purecloud.apps.ClientApp({
-                pcEnvironment,
-            });
-            Vue.prototype.$clientApp = clientApp;
+            if (queryParameters.gcHostOrigin && queryParameters.gcTargetEnv) {
+                clientApp = new window.purecloud.apps.ClientApp({
+                    gcHostOrigin: queryParameters.gcHostOrigin,
+                    gcTargetEnv: queryParameters.gcTargetEnv
+                });
+                pcEnvironment = clientApp.pcEnvironment;
+                Vue.prototype.$clientApp = clientApp;
+            } else if (pcEnvironment) {
```

Comment:
eventually, we can kill off support for pcEnvironment in the demo apps.  But, smart to leave it in for now.  Until we update all config using them.